python/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2018 The Python Packaging Authority
```

Comment:
Should this be LastMIle?

Also I guess this is MIT licensed?

Reply:
Will update, and yes it's the MIT license